spock-core/src/main/java/org/spockframework/mock/runtime/ByteBuddyMockFactory.java

```diff
@@ -83,6 +83,26 @@ class ByteBuddyMockFactory {
     }
   }
 
+  /**
+   * {@return true, if the mock is considered local}
```

Comment:
This syntax seams strange.
Shouldn't the `@return` a the parameters without the `{}`?

Reply:
Yeah thats a Java 16 feature. Forgot that we are targeting 8 here, so I'll change it :)